cleverpush/src/main/java/com/cleverpush/util/BroadcastReceiverUtils.java

```diff
@@ -0,0 +1,92 @@
+package com.cleverpush.util;
+
+import static com.cleverpush.CleverPush.broadcastReceiverHandler;
+import static com.cleverpush.CleverPush.context;
+
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+
+import com.cleverpush.CleverPush;
+import com.cleverpush.CleverPushPreferences;
+import com.cleverpush.Constants;
+
+import java.util.UUID;
+
+public final class BroadcastReceiverUtils {
+    /**
+     * This function will register broadcast receiver with intent if only channel has preventDuplicateEnabled in config or if it's not already registered
+     *
+     * @param context
+     * @function registerReceiver
+     */
+    public static void registerReceiver(CleverPush context) {
```

Comment:
dont call it context since that word has a different meaning in Android i think, maybe just call it cleverPushInstance @gbhrdt other name?

Reply:
![image](https://user-images.githubusercontent.com/104719696/195083249-963c9f3e-fe3a-4df6-a5bd-c0f76a4d4420.png)
is it okay @cpKevinrodriguesnobre @gbhrdt?

Reply:
![image](https://user-images.githubusercontent.com/104719696/195083453-8331488d-0964-4f35-bc12-00974553a171.png)
okay added Please check @cpKevinrodriguesnobre @gbhrdt

Reply:
cleverPushInstance is good